source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2598,6 +2647,9 @@ export class HTML5MediaPlayer extends MediaPlayer {
         );
         this._mediaElement.setAttribute("webkit-playsinline", "");
         this._mediaElement.setAttribute("playsinline", "");
+        // I had to put this here for testing because the caption file has a different domain as the video and html doesn't like that.
+        // We should probably remove this before merging
+        this._mediaElement.setAttribute("crossorigin", "");
```

Comment:
You may need to remove this in code review

Reply:
I left it in so that reviewers can test it and see it working

Reply:
don't forget to remove test pieces :)

Reply:
Updated the comment, we will leave crossorigin enabled